src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AnnotationsCacheWarmer.php

```diff
@@ -76,6 +77,16 @@ protected function doWarmUp($cacheDir, ArrayAdapter $arrayAdapter)
         return true;
     }
 
+    protected function warmUpPhpArrayAdapter(PhpArrayAdapter $phpArrayAdapter, array $values)
+    {
+        $values = array_filter($values, function ($val) {
+            return null !== $val;
+        });
+
+        // make sure we don't cache null values
```

Comment:
could be moved before the call to array_filter (same below)

Reply:
Fixed